### PR TITLE
Makes ATMs put money into your hands when withdrawing

### PR DESCRIPTION
**Pull request by SabreML**
## What Does This PR Do

This makes ATMs try to put money directly into your hands when withdrawing. As opposed to just dropping it on the floor next to you.

Also added some simple instructions to the ATM description, because why not.

## Why It's Good For The Game

Don't want people stealing your incredibly valuable money.

## Changelog
:cl:
tweak: Withdrawing money from an ATM now puts it into your hands if possible, rather than just dropping it onto the floor
tweak: Tweaked the ATM description to be a bit more user-friendly
/:cl:

### CODE REVIEW

**Review by Kyep — APPROVED** [review]

File: code/modules/economy/ATM.dm [hunk 1]
```diff
@@ -301,4 +301,6 @@ log transactions
 
 //create the most effective combination of notes to make up the requested amount
 /obj/machinery/atm/proc/withdraw_arbitrary_sum(arbitrary_sum)
-	new /obj/item/stack/spacecash(get_step(get_turf(src), turn(dir, 180)), arbitrary_sum)
+	var/obj/item/stack/spacecash/C = new(amt = arbitrary_sum)
+	if(!usr?.put_in_hands(C))
```
Comment (farie82) [inline code/modules/economy/ATM.dm:305]: this will also magically teleport cash into the hands of a TK user

**Review by farie82 — CHANGES_REQUESTED** [review]

File: code/modules/economy/ATM.dm (hunk 1 above)
Comment (SabreML) [reply code/modules/economy/ATM.dm:305]: damn snowflake tk

File: code/modules/economy/ATM.dm [hunk 2]
```diff
@@ -16,7 +16,7 @@ log transactions
 
 /obj/machinery/atm
 	name = "Nanotrasen automatic teller machine"
-	desc = "For all your monetary needs!"
+	desc = "For all your monetary needs! Just insert your ID card to make a withdrawl or deposit!"
```
Comment (dearmochi) [inline code/modules/economy/ATM.dm:19]:
```suggestion
	desc = "For all your monetary needs! Just insert your ID card to make a withdrawal or deposit!"
```

File: code/modules/economy/ATM.dm [hunk 3]
```diff
@@ -16,7 +16,7 @@ log transactions
...
 	icon = 'icons/obj/terminals.dmi'
 	icon_state = "atm"
 	anchored = 1
```
Comment (dearmochi) [inline code/modules/economy/ATM.dm:22]:
```suggestion
	anchored = TRUE
```

File: code/modules/economy/ATM.dm (hunk 2 above)
Comment (SabreML) [reply code/modules/economy/ATM.dm:19]: Huh, never knew it was spelled that way

**Pull request merged**